nodejs/week2/homework/meal-sharing/src/backend/api/reservations-router.js

```diff
@@ -0,0 +1,38 @@
+const express = require("express");
+const router = express.Router();
+
+const reservations = require("./../data/reservations.json");
+router.get("/", async (request, response) => {
+    try {
+        response.send(reservations);
+    } catch (error) {
+        throw error;
```

Comment:
throwing an error here is not helpful because it would crash your server app if it's not caught.
you can return an error code like `500 Internal Server Error` instead

Reply:
I removed the try catch